test/test_torch.py

```diff
@@ -5832,35 +5846,35 @@ def test_pow(self, device):
                     res1 = torch.pow(m1[4], num)
                     res2 = res1.clone().zero_()
                     for i in range(res2.size(0)):
-                        res2[i] = math.pow(m1[4][i], num)
-                    self.assertEqual(res1, res2)
+                        res2[i] = pow(m1[4][i], num)
+                    self.assertAlmostEqual(res1, res2, places=4)
```

Comment:
This is a reduction of precision for non-complex types?

Reply:
for complex dtypes, assertEqual fails because there's a difference of 3 * 10 ^ -5 > 10 ^ -5

Reply:
I guess my point is, you've reduced the precision of this test for non complex, perhaps in an unwarranted way.